databases/week3/meal_sharing_db.sql

```diff
@@ -0,0 +1,48 @@
+CREATE DATABASE IF NOT EXISTS meal_sharing;
+
+USE meal_sharing;
+
+CREATE TABLE meal(
+                     id INT(11) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+                     title VARCHAR(255) NOT NULL,
+                     description TEXT NULL DEFAULT NULL,
+                     location VARCHAR(255) NOT NULL,
+                     meal_time DATETIME NULL DEFAULT NULL,
+                     max_reservations INT NOT NULL,
+                     price DECIMAL(10,2) NOT NULL,
+                     created_date DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP,
+                     updated_date DATETIME NULL ON UPDATE CURRENT_TIMESTAMP
+)  ENGINE=INNODB DEFAULT CHARSET=UTF8MB4 COLLATE = UTF8MB4_UNICODE_CI;
+CREATE INDEX location
+    ON meal (location);
```

Comment:
What is the advantage of creating an index here..? Many cool advantages...
`Indexes are used to retrieve data from the database more quickly than otherwise. The users cannot see the indexes, they are just used to speed up searches/queries.` 
**Note: Updating a table with indexes takes more time than updating a table without (because the indexes also need an update)**